front/src/modules/trainschedule/components/Timetable/TimetableToolbar.tsx

```diff
@@ -71,65 +72,93 @@ const TimetableToolbar = ({
     useState<ScheduledPointsHonoredFilter>('both');
   const [selectedTags, setSelectedTags] = useState<Set<string | null>>(new Set());
 
+  const [totalPacedTrainCount, totalTrainScheduleCount] = timetableItems.reduce(
+    (acc, { id }) => {
+      if (isTrainSchedule(id)) {
+        acc[1] += 1;
+      } else {
+        acc[0] += 1;
+      }
+      return acc;
+    },
+    [0, 0]
+  );
```

Comment:
It's better to name what you are handling. You can also add a useMemo.

```suggestion
  const { totalPacedTrainCount, totalTrainScheduleCount } = timetableItems.reduce(
    (acc, { id }) => {
      if (isTrainSchedule(id)) {
        acc.totalTrainScheduleCount += 1;
      } else {
        acc.totalPacedTrainCount += 1;
      }
      return acc;
    },
    {totalPacedTrainCount: 0, totalTrainScheduleCount: 0}
  );
```

Reply:
Done